### ISSUE TITLE
[a11y-defect-x]: Information, structure, and relationships found in 508 Audit on form 21-4142

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#63435]
### Point of contact

Jeana Clark

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

...
- Information, structure, and relationships conveyed through presentation cannot be understood (programmatically determined). 
- Staging URL - https://staging.va.gov/supporting-forms-for-claims/release-information-to-va-form-21-4142
- 

### Reproduction steps

1.On “Step 12 of 12: Review application” screen, each collapsed item is read as “list of 1 items”. See Figure 3 below. 

On “Step 12 of 12: Review application” screen, when you expand the Personal Information and press the arrow key, “definition list of 5 items nesting level 1t of 1 items” is read by assistive technology. When you press the arrow key again, the information starting with First name: is read as unstructured text by assistive technology.
...

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by tbaker1026** [comment]
![](https://github.com/department-of-veterans-affairs/va.gov-team/assets/115116356/d0d92223-3130-4593-bbf0-12f1102cf88a)

![](https://github.com/department-of-veterans-affairs/va.gov-team/assets/115116356/501d685d-8fc4-4e3a-9b85-87c0f6042c75)

**Comment by jeana-adhoc** [comment]
When I asked for more clarification about this problem, this was the response from the 508 office:

> You have a bunch of items that are expand tags.   And you have each of them defined as a standalone lists so you have 9 different lists of one item each.  Then when you open up each of those items you have a sub-list of items that are not structured. That is very confusing trying to listen to that with a screen reader.   Either remove the lists surrounding the expand tags so they aren’t lists or have all the expand tags in a single list.   

> So For consistency sake if you put the 9 expand tags in a lists then when you expand the tag, the contents inside should also be a lists.  If the expand tags are not in a list then the information inside them can remain unstructured.
